hbase-server/src/test/java/org/apache/hadoop/hbase/master/http/TestMasterStatusServlet.java

```diff
@@ -1,152 +0,0 @@
-/*
```

Comment:
Since this test class is removed, how do we test the new jsp page?

Reply:
Thanks, this is a very good question. :+1: 

As far as I understand before we only called the `render()` method and maybe set some parameters on the Jamon template but we did not verified the actual HTML page content.

I can think of a test to call the master status page, maybe parse the response and then do some assertions on the returned HTML.
 
Does this sound useful?

Reply:
Sound good. Even no assertions, at least we can make sure that there is no exception while rendering the page.

Reply:
OK, thanks for your response. :+1:  Then I'll add that test.

As I see, the UI code got updated on master so then I'll have to update this branch with those changes.

Reply:
OK, I added a new test (`TestMasterStatusPage`) which does what we mentioned and asserts some basic parts of the page.